db/src/main/resources/sql/flyway/V108__add_host_indexes_for_whois.sql

```diff
@@ -12,6 +12,5 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 
-
-CREATE INDEX IDXovmntef6l45tw2bsfl56tcugx ON "Host" (deletion_time);
```

Comment:
Why did you remove the `deletion_time` index?  It's still going to be useful to have to be able to discern between hosts that are active and those that are not (even if for ad hoc queries and analytics if nothing else).  Not having an index on the column that is used to determine whether or not a given entity is deleted or not in a database that uses a soft-deletion paradigm seems like a mistake.

Reply:
Query analyzer says that the index on deletion_time is not used in this and similar queries.

I believe the only use cases where an index on deletion_time is useful are:

1. Specific time is given: deletion_time = ...
2. Deletion time is the only parameter: .. where deletion_time > ...;

Otherwise the cardinality of the intermediate result is usually too large to be useful.